modules/atomic-global-variables/classes/hooks.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Elementor\Modules\AtomicGlobalVariables\Classes;
+
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+use Elementor\Modules\AtomicGlobalVariables\PropTypes\Color_Variable_Prop_Type;
+use Elementor\Modules\AtomicGlobalVariables\Transformers\Global_Variable as Global_Variable_Transformer;
+use Elementor\Modules\AtomicWidgets\PropsResolver\Transformers_Registry;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class Hooks {
+	private Wordpress_Adapter_Interface $wp_kernel;
```

Comment:
I assume it is important that we implement this in a consistent way in the whole codebase. Wdyt?

Reply:
@mike-elementor Yes, I believe it is important that we have a general naming convention.